templates/example_cell_clustering.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Again, can we save this to csv? For Zak's tool and to make it easier for user's to create their own heatmaps if they want.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>See my comment in the pixel clustering notebook. Same applies here. I think it would be better to first run hierarchical clustering, then make the heatmap then save the table.</p>

Reply:
So as a heads up, with the new workflow of adding in Zak's interactive re-clustering, it'll be easier to update the workflow to save before visualizing.